browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -223,7 +223,10 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			if (handler)
 				handler(parentContainer, data, builder);
 			else if (data.text || data.command || data.postmessage) {
+				const originalNoLabelsForUnoButtons = builder.options.noLabelsForUnoButtons;
+				builder.options.noLabelsForUnoButtons = data.noLabel ? data.noLabel : originalNoLabelsForUnoButtons;
```

Comment:
I'm not sure about that: `noLabel` if exists it will be passed in `data` to the `unoToolButton` handler.
We should do  that inside. Why: we can get update of single unoToolButton (no parent). It has to work in that case too :)

Reply:
please do what is needed but not here, please check inside unotoolbutton